src/Pages/ErrorPage/index.js

```diff
@@ -25,25 +25,25 @@ function ErrorPage() {
   }, []);
 
   return (
-    <div className={styles.pageContainer}>
+    <div className={styles.container}>
       {isAuthenticated() && <Navbar />}
+
       <div className={styles.body}>
-        <div className={styles.leftBody}>
+        <div className={styles.bodyContent}>
           <h1 className={styles.heading}>404</h1>
           <p className={styles.subhead}>Looks like this page went bananas.</p>
           <div className={styles.buttonContainer}>
             <Button type="submit" variant="buttonSecondary" text="Go back" action={goBack} />
```

Comment:
@tackc looks like on the figma example they removed the button borders with `border: none`

Reply:
<img width="1424" alt="image" src="https://github.com/FoodIsLifeBGP/banana-admin/assets/13320440/bff51f01-e32a-46a6-ab72-c7864fc7fff6">

Reply:
Sure enough! I totally missed that! Thanks...it's resolved 😀